changelog.d/0-release-notes/pr-2035

```diff
@@ -0,0 +1 @@
+Backend now separates conversation access control for guests and services. The old access roles are still supported but it is encouraged to upgrade clients since mapping between the old access roles and the new access roles are is not isomorphic. For more details refer to the API changes changelog, the Swagger docs, or the technical specification.
```

Comment:
If we already know which versions of clients support this, we should add those here, but I suspect we're not certain yet.  Bring up in the next standup?